cdl_examples/tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803.cdl

```diff
@@ -0,0 +1,90 @@
+netcdf tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803 {
+dimensions:
+	axis_nbounds = 2 ;
+	x = 567 ;
+	y = 264 ;
+	deptht = 75 ;
+	time = UNLIMITED ; // (1 currently)
+variables:
+	float nav_lat(y, x) ;
+		nav_lat:standard_name = "latitude" ;
+		nav_lat:long_name = "Latitude" ;
+		nav_lat:units = "degrees_north" ;
+	float nav_lon(y, x) ;
+		nav_lon:standard_name = "longitude" ;
+		nav_lon:long_name = "Longitude" ;
+		nav_lon:units = "degrees_east" ;
+	float deptht(deptht) ;
+		deptht:name = "deptht" ;
+		deptht:long_name = "Vertical T levels" ;
+		deptht:units = "m" ;
+		deptht:positive = "down" ;
+		deptht:bounds = "deptht_bounds" ;
+	float deptht_bounds(deptht, axis_nbounds) ;
+		deptht_bounds:units = "m" ;
+	double time(time) ;
+		time:axis = "T" ;
+		time:standard_name = "time" ;
+		time:long_name = "Time axis" ;
+		time:calendar = "gregorian" ;
+		time:units = "days since 1900-01-01 00:00:00" ;
+		time:time_origin = "1900-01-01 00:00:00" ;
+		time:bounds = "time_bounds" ;
+	double time_bounds(time, axis_nbounds) ;
+	float tpoce(time, deptht, y, x) ;
+		tpoce:standard_name = "sea_water_potential_temperature" ;
+		tpoce:long_name = "Sea Water Potential Temperature" ;
+		tpoce:units = "degC" ;
+		tpoce:online_operation = "average" ;
+		tpoce:cell_methods = "time: mean" ;
+		tpoce:interval_operation = "720 s" ;
+		tpoce:interval_write = "1 month" ;
+		tpoce:_FillValue = 1.e+20f ;
+		tpoce:missing_value = 1.e+20f ;
+		tpoce:coordinates = "nav_lat nav_lon" ;
+		tpoce:description = "None" ;
+		tpoce:history = "none" ;
+		tpoce:cell_measures = "area: areacello" ;
+
+// global attributes:
+		:name = "/scratch/work/sevault/CNRM-RCSM6B/RCSM6B-ERA5.2/outputs/19880101/tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_%start_date%-%end_date%" ;
+		:description = "Created by xios" ;
+		:creation_date = "2024-03-29T07:43:23Z" ;
+		:tracking_id = "hdl:21.14103/227c3a13-dde3-4b47-987e-3d7e0120cae1" ;
+		:title = "CNRM-RCSM6B-SN model output prepared for CORDEX-CMIP6 / evaluation simulation driven by ERA5, evaluation, r1i1p1f1" ;
+		:Conventions = "CF-1.10" ;
+		:activity_id = "RCM" ;
+		:comment = "CORDEX MED-12 CNRM-RCSM6B-SN L91." ;
+		:contact = "contact.aladin-cordex@meteo.fr" ;
+		:dr2xml_version = "3.0" ;
+		:domain = "MED-12" ;
+		:domain_id = "MED-12" ;
+		:driving_experiment = "ERA5, evaluation, r1i1p1f1" ;
```

Comment:
driving_experiment is not free to choose, see https://github.com/WCRP-CORDEX/cordex-cmip6-cv/blob/main/CORDEX-CMIP6_driving_experiment_id.json

In general, all attributes in the CV must follow exactly the values there https://github.com/WCRP-CORDEX/cordex-cmip6-cv